src/winetricks

```diff
@@ -14827,6 +14827,54 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Microsoft Powershell" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="ProtectionId.685.December.2016.rar" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/protection_id.exe"
+
+load_powershell()
+{
+	# Requires at least .NET Framework 4.5.2 (https://docs.microsoft.com/en-us/powershell/scripting/wmf/setup/install-configure?view=powershell-6)
+	w_call dotnet462
+
+	# Download 64-bit WMF (Windows Management Framework)
+	w_download "http://download.microsoft.com/download/6/F/5/6F5FF66C-6775-42B0-86C4-47D41F2DA187/Win7AndW2K8R2-KB3191566-x64.zip"
```

Comment:
This should only occur for 64-bit prefixes.

Reply:
Like making WMF verb for 32-bit and 64-bit ?